tmpfiles.d/meson.build

```diff
@@ -33,7 +33,8 @@ in_files = ['etc.conf',
 
 foreach file : in_files
         custom_target(
-                file,
+                # XXX: workaround for old meson. Drop when upgrading.
```

Comment:
maybe add version info to this comment? and a FIXME tag?

Reply:
`XXX` is like `FIXME`, no? (And it's not really FIXME… It's something ugly but beyond our control for now. So I think XXX is better than FIXME).  I have no idea in which version exactly this was fixed, and I don't want to do meson git archaeology to find out. In the commit message I wrote that it reproduces with the oldest meson version I could easily install, 0.49.2 from buster.

Reply:
lgtm.com picks up FIXME as a magic word and tracks it as things to fix. (Doesn't coverity too?) Are you sure it also picks up XXX?

Reply:
I don't think coverity does anything about either of the two.
Do we make any use of the tracking in lgtm.com? I have noticed the comments from coverity, and it could be useful to notice that a pull request has a fixme, if the fixme was intended to be fixed before the pull request is merged. Here, if lgtm made a comment on the pull request, it would be just noise.

Reply:
gnome-builder picks up both FIXME + XXX.

I did look at the lgtm FIXME list every now and then. It's moderately useful I'd say